.vscode/settings.json

```diff
@@ -1 +1,3 @@
-{}
+{
```

Comment:
Was this change meant to be for the repo?
Because you can apply it on your user profile level in VSCode instead of project level.

Reply:
No it is out. 👍

Reply:
Then you no longer require review from Core :)

Reply:
Ok, thanks. And sorry for the inconvenience.